core/tracing/CHANGELOG.md

```diff
@@ -0,0 +1,69 @@
+# Changelog
+
+All notable changes to the tracing interface will be documented in this file.
```

Comment:
Heh, that's quite a promise you're making here :)

Reply:
:trollface: